docs/modules/ROOT/pages/index.adoc

```diff
@@ -47,6 +47,151 @@ If you want to use the Bundle generator, you will first need to use Quarkus 2.3.
+
+So far, we have already built the Operator bundle image at `$MY_BUNDLE_IMAGE` (see above) and next, we need to add this Operator bundle image into our catalog. For doing this, we'll use the `olm` tool as follows:
+
+[source,shell script]
+----
+CATALOG_IMAGE=<catalog container registry>/<catalog container registry namespace>/<catalog name>:<tag>
+opm index add \
+    --bundles $MY_BUNDLE_IMAGE \
+    --tag $CATALOG_IMAGE \
+    --build-tool docker
+docker push $CATALOG_IMAGE
+----
+
+For example, if our catalog name is `my-catalog`, our container registry for the catalog is `quay.io`, our Quay user is `myuser` and the container tag we're releasing is `59.0`, the final `CATALOG_IMAGE` property would be `quay.io/myuser/my-catalog:59.0`.
+
+[NOTE]
+====
+If you're using an insecure registry, you'd need to append the argument `--skip-tls` to the `opm index` command.
+====
+
+Once we have our catalog image built and pushed at `$CATALOG_IMAGE` image, we need to install it on our cluster using the `CatalogSource` resource by doing the next command:
+
+[source,shell script]
+----
+cat <<EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: my-catalog-source
+  namespace: operators
+spec:
+  sourceType: grpc
+  image: $CATALOG_IMAGE
+EOF
+----
+
+[NOTE]
+====
+We need to install the catalog source at the Kubernetes namespace where we installed OLM, which by default is `operators`.
+====
+
+Once the catalog is installed, you should see the catalog pod up and running:
+
+[source,shell script]
+----
+kubectl get pods -n <K8S NAMESPACE> --selector=olm.catalogSource=my-catalog-source
+----
+
+4. Install your operator via OLM
+
+OLM deploys operators via https://olm.operatorframework.io/docs/tasks/install-operator-with-olm/#install-your-operator[subscriptions]. Creating a  `Subscription` will trigger the operator deployment. You can simply create the `Subscription` resource that contains the operator name and channel to install by running the following command:
+
+[source,shell script]
+----
+cat <<EOF | kubectl create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: my-subscription
+  namespace: <Kubernetes namespace where your operator will be installed>
+spec:
+  channel: alpha
+  name: my-operator-name
+  source: my-catalog-source
+  sourceNamespace: operators
+EOF
+----
+
+[NOTE]
+====
+We'll install the operator in the target namespace defined at the metadata object. The `sourceNamespace` value is the Kubernetes namespace where the catalog was installed on (`operators` by default, see the previous step).
```

Comment:
```suggestion
We'll install the operator in the target namespace defined in the metadata object. The `sourceNamespace` value is the Kubernetes namespace where the catalog was installed on (`operators` by default, see the previous step).
```

The `$OLM_NAMESPACE` variable should be used here as well, if we decide to introduce it.